package.json

```diff
@@ -11,7 +11,23 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "stylelint": "stylelint **/*.css",
-    "stylelint:fix": "stylelint --fix **/*.css"
+    "stylelint:fix": "stylelint --fix **/*.css",
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```

Comment:
Could you please move the configuration to `lint-staged.config.js` that should be added to the project root? I want to have cleaner and more readable `package.json`.

Reply:
I moved configuration to **`lint-staged.config.js`** file.